template/README.md

```diff
@@ -20,36 +20,47 @@ The vagrant box ships with a default startup scheme. `make` from this directory
 You may change the hashistack configuration or add aditional pre and post steps to the startup procedure to match your needs.
 Detailed documentation in [dev/vagrant/conf/README.md](dev/vagrant/conf/README.md)
 
-### Nomad ACLs
+### Pre packaged configuration switches
 
-| default   | environment variable  |  value  |
-|:---------:|:----------------------|:-------:|
-|           | nomad_acl             |  true   |
-| x         | nomad_acl             |  false  |
+The box comes standard with a set of environment switches to simplify testing of different scenarios and enable staged development efforts.
 
```

Comment:
```suggestion

#### Enterprise vs Open Source Software (OSS)
As long as Enterprise is not set to `true` the box will utilise OSS version of the binaries. 
```

Reply:
Did you mean to remove `The box comes standard with a set of environment switches to simplify testing of different scenarios and enable staged development efforts.` ?

Reply:
See 5972815

Reply:
Sorry for late answer, but yeah, last commit is exactly what I tried to suggest!